sdk/eventgrid/eventgrid/swagger/README.md

```diff
@@ -17,15 +17,27 @@ output-folder: ../
 save-inputs: true
 source-code-folder-path: ./src/generated
 use-extension:
-  "@autorest/typescript": "6.0.0-dev.20210121.2"
+  "@autorest/typescript": "1.0.0-beta.2"
```

Comment:
I would suggest to replace this with `typescript: true` so you always get the most recent version of the TS extension.

Reply:
Thanks, @deyaaeldeen, I think when I prototyped this a few weeks ago we hadn't cut over yet and so I needed the explicit version, but glad I can get out of this "pin to a specific" version game.

I made the change and re-ran the generator (and it even brought in some code-gen changes since I was on an older version).